static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -113,6 +113,10 @@ export const shouldIncludeAdYouLike = (slotSizes: PrebidSize[]): boolean =>
 export const shouldIncludeOzone = (): boolean =>
     !isInUsRegion() && !isInAuRegion();
 
+// TODO: Check is we want regional restrictions on where we load the ozoneBidAdapter
+export const shouldUseOzoneAdaptor = (): boolean =>
+    !isInUsRegion() && !isInAuRegion() && config.get('switches.prebidOzone');
```

Comment:
I think the switch test is done automatically, which is why you add the switchName when you set up the bidder at https://github.com/guardian/frontend/pull/21251/files#diff-3504a8960b869185b559c4491c6a5857R372

Reply:
Good point! I will remove it in the next iteration. 

Plan is to get this on PROD but keep it switched off so I can do some testing with the pbtest params, and better communicate the issue externally.